Sources/MarathonCore/ScriptManager.swift

```diff
@@ -59,7 +59,7 @@ internal final class ScriptManager {
 
     // MARK: - API
 
-    func script(at path: String) throws -> Script {
+    func makeScript(at path: String) throws -> Script {
```

Comment:
Not sure if I agree that this should be called `makeScript`, since it might not actually end up **making** a script (if one alreay exists, it's simply returned.